functional-tests/src/main/scala/com/gu/membership/pages/EventsListPage.scala

```diff
@@ -6,22 +6,14 @@ import org.openqa.selenium.{JavascriptExecutor, By, WebDriver, WebElement}
 
 class EventsListPage(driver: WebDriver) extends BaseMembershipPage(driver) {
 
-  private def eventLinkList: util.List[WebElement] = driver.findElements(By.cssSelector(".event-item"))
+  private def eventLinkList: util.List[WebElement] = driver.findElements(By.cssSelector(".qa-available-event-item"))
 
-  private def eventsTitleList: util.List[WebElement] = driver.findElements(By.cssSelector(".event-item__title>span"))
-
-  private def eventsLocationList: util.List[WebElement] = driver.findElements(By.cssSelector(".event-item__location"))
```

Comment:
This wasn't a selector used anywhere in the markup and `getEventLocationByIndex` doesn't seem to be used anywhere so I've removed this.